sdcm/sct_events.py

```diff
@@ -63,16 +60,35 @@ def run(self):
             backend.close()
             context.term()
 
-    def subscribe_events(self, filter_type=b'', stop_event=None):
-        # pylint: disable=too-many-nested-blocks
+    @staticmethod
+    @retrying(n=0, timeout=120)
```

Comment:
Do you use `@retrying` as a timeout mechanism? If so define `@time_out(t)`
by using retrying and `functools.partial`

Reply:
Done

Reply:
I don't see it is done...